src/ghost/ghost-post.service.integration.test.ts

```diff
@@ -160,42 +160,6 @@ describe('GhostPostService', () => {
             );
         });
 
-        it('should create a new post when the post does not exist', async () => {
```

Comment:
Tested [here](https://github.com/TryGhost/ActivityPub/pull/1073/files#diff-5c81e24007c2cfda5c5b7b00fc25c87538a30c855661cbe153797a7b8cc6c4a2R181) in unit tests. And we already have integration tests for creating new post so this feels redundant.

Reply:
Yes, as long as we do have one integration test that tests that creating a new post works - then I think it's ok to test the failure cases in unit tests